<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>63</y>
     <width>524</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QDockWidget" name="uiToolDock" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="windowTitle" >
         <string/>
        </property>
        <widget class="QWidget" name="dockWidgetContents" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>21</y>
           <width>71</width>
           <height>360</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="uiView" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenImage" />
    <addaction name="separator" />
    <addaction name="actionLogin" />
    <addaction name="actionNewTemplate" />
    <addaction name="actionOpenTemplate" />
    <addaction name="actionSaveTemplate" />
    <addaction name="actionSaveTemplateAs" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionToggleImageBoxes" />
    <addaction name="actionToggleTemplateBoxes" />
    <addaction name="actionToggleBinarized" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionDelete" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>464</y>
     <width>524</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>30</y>
     <width>524</width>
     <height>33</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleImageBoxes" />
   <addaction name="actionToggleTemplateBoxes" />
   <addaction name="actionToggleBinarized" />
   <addaction name="separator" />
   <addaction name="actionUnzoom" />
   <addaction name="actionZoom" />
  </widget>
  <action name="actionOpenImage" >
   <property name="text" >
    <string>&amp;Open Image</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionToggleImageBoxes" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Show Image Boxes</string>
   </property>
  </action>
  <action name="actionLogin" >
   <property name="text" >
    <string>&amp;Login</string>
   </property>
  </action>
  <action name="actionSaveTemplate" >
   <property name="text" >
    <string>&amp;Save Template</string>
   </property>
  </action>
  <action name="actionOpenTemplate" >
   <property name="text" >
    <string>Open &amp;Template</string>
   </property>
  </action>
  <action name="actionNewTemplate" >
   <property name="text" >
    <string>New Template</string>
   </property>
  </action>
  <action name="actionToggleTemplateBoxes" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Template Boxes</string>
   </property>
  </action>
  <action name="actionToggleBinarized" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Show Binarized</string>
   </property>
  </action>
  <action name="actionSaveTemplateAs" >
   <property name="text" >
    <string>Save Template &amp;As...</string>
   </property>
  </action>
  <action name="actionDelete" >
   <property name="text" >
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut" >
    <string>Del</string>
   </property>
  </action>
  <action name="actionZoom" >
   <property name="text" >
    <string>&amp;Zoom</string>
   </property>
  </action>
  <action name="actionUnzoom" >
   <property name="text" >
    <string>Unzoom</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
